shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -180,6 +183,18 @@ - (BOOL)setupShell {
   return true;
 }
 
+- (BOOL)isViewOpaque {
+  return _isViewOpaque;
+}
+
+- (void)isViewOpaque:(BOOL)value {
+  _isViewOpaque = value;
+  if (_flutterView.get().layer.opaque != value) {
+    _flutterView.get().layer.opaque = value;
+    [_flutterView.get().layer setNeedsLayout];
```

Comment:
Doesn't compile.

Reply:
Oh, right, because in this case `layer` is a member of an Objective-C object (`_flutterView.get().layer` is actually `[_flutterView.get() layer]`).  Oops.  Carry on.

Reply:
I can't tell you how many times a day I'm forgetting Objective C syntax and messing stuff like that up ;)